torch/_dynamo/variables/ctx_manager.py

```diff
@@ -181,6 +181,14 @@ def exit(self, tx, *args):
         self._call_func(tx, self.initial_values)
         return variables.ConstantVariable.create(None)
 
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ):
+        # TODO(jon-chuang): uncomment once https://github.com/pytorch/pytorch/issues/113298 is fixed
+        # self._call_func(tx, self.initial_values)  # undo eager initialization
+        self.initialized = False
```

Comment:
Mutating variable trackers in unsafe because we make copies of them frequently.

Reply:
Actually, for now, I think we don't need to mutate anything. We may need to if and when https://github.com/pytorch/pytorch/issues/113298 is fixed.

Reply:
removed the initialized var in favour of statelessly always running `_call_func`